README.adoc

```diff
@@ -16,39 +16,57 @@ link:https://access.redhat.com/containers/#/registry.access.redhat.com/openjdk/o
 
 You need to https://cekit.readthedocs.io/en/develop/installation.html[install Cekit] to build these images.
 
+These sources are prepared and tested for Cekit 3.6.0.
+
 ## Image versions
 
-### RHEL 7 JDK 8
+### RHEL 7 OpenJDK 8
 
-```
-cekit build
-```
+    cekit --redhat build podman
 
 This will produce an image named `redhat-openjdk-8/openjdk18-openshift`
 
-### RHEL 7 JDK 11
+### RHEL 7 OpenJDK 11
 
-```
-cekit --descriptor openjdk-11-rhel7.yaml build
-```
+    cekit --redhat --descriptor openjdk-11-rhel7.yaml build podman
 
 This will produce an image named `openjdk/openjdk-11-rhel7`
 
-### RHEL 8 JDK 8
+### UBI 8 JDK 8
```

Comment:
For consistency, this should probably be `OpenJDK 8`. I know this is a merge, so feel free to do this in develop and pick it up with the next merge.